alpha/experience-platform-extension/schema-creation.md

```diff
@@ -0,0 +1,191 @@
+
+The [Experience Event Commerce Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-commerce.schema.md) mixin defines a `Commerce` object and a `ProductListItemsItem` object. The `Commerce` object lets you specify which actions are happening in a list of `ProductListItemsItem`s. 
+
+### Commerce actions
+
+Each action object that is defined in the `Commerce` object takes a value, typically set to 1, which indicates that action occurred.
+
+- CartAbandons - A product list has been identified as no longer accessible (for example purchasable) by the user.
+- Checkouts - An action during a checkout process of a product list.
+- ProductListAdds - Addition of a product to the product list.
+- ProductListOpens - Initializations of a new product list.
+- ProductListRemovals - Removal of a product entry from a product list.
+- ProductListReopens - A product list that was not accessible(abandoned) and has been reactivated by the user.
+- ProductListViews - Views of a product-list has occurred.
+- ProductViews - Views of a product have occurred.
+- Purchases - An order that has been accepted.
+- SaveForLaters - Product list that is saved for future use.
+
+Here is a sample of creating a `Commerce` object with the `ProductListAdds` action:
+
+{% tabs %}
+{% tab title="Android" %}
+
+```java
+Commerce commerce = new Commerce();
+ProductListAdds productListAdds = new ProductListAdds();
+productListAdds.setValue(1);
+commerce.setProductListAdds(productListAdds);
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Commerce order
+
+The `Commerce` object also contains details about a purchase through the `Order` object. The `Order` object is typically included with a `Purchases` action and includes details for purchase ID and order number, total price, currency, and a list of payment items.
+
+Here is a sample of creating a `Commerce` object with an `Order`:
+
+{% tabs %}
+{% tab title="Android" %}
+
+```java
+// Create PaymentItem which details the method of payment
+PaymentsItem paymentsItem = new PaymentsItem();
+paymentsItem.setCurrencyCode("USD");
+paymentsItem.setPaymentAmount(25.99);
+paymentsItem.setPaymentType("credit_card");
+
+List<PaymentsItem> paymentsItemList = new ArrayList<>();
+paymentsItemList.add(paymentsItem);
+
+// create the Order
+Order order = new Order();
+order.setCurrencyCode("USD");
+order.setPriceTotal(25.99);
+order.setPayments(paymentsItemList);
+
+// create Commerce and add Purchases action and Order details
+Commerce commerce = new Commerce();
+Purchases purchases = new Purchases();
+purchases.setValue(1);
+commerce.setPurchases(purchases);
+commerce.setOrder(order);
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Product list items
+
+The `ProductListItemsItem` object includes details for a product that was selected by the customer. A list of  `ProductListItemsItem` objects can be used to identity the products with which the customer has interacted.
```

Comment:
Please reword the 2nd sentence to:
"The following `ProductListItemsItem` objects can be..."